OneCommunityGlobal/HGNRest
# yihan create editable four-letter team code option

# Description
<img width="746" alt="image" src="https://github.com/OneCommunityGlobal/HGNRest/assets/94303659/be49c5bf-89ef-43a8-945c-10eccbd66c09">

## Related PRS (if any):
To test this backend PR you need to checkout the [#1162](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1162) frontend PR.

## Main changes explained:
- Update file userProfile.js and userProfileController.js for adding teamCode property to userProfile object.
- Update file reporthelper.js for adding teamCode property to weekly summaries report object.

## How to test:
1. check into current branch
2. do `npm run build` and `npm start` to run this PR locally
3. follow test steps in front end PR #XXX
